sdk/workflow.go

```diff
@@ -183,6 +183,26 @@ type WorkflowNode struct {
 	Triggers         []WorkflowNodeTrigger `json:"triggers,omitempty" db:"-"`
 }
 
+//EqualsTo returns true ia a node has the same pipeline, and context than another
```

Comment:
// EqualsTo returns true if a node has the same pipeline and context than another